oshi-core/src/main/java/oshi/util/ExecutingCommand.java

```diff
@@ -81,7 +83,16 @@ private ExecutingCommand() {
     public static List<String> runNative(String[] cmdToRunWithArgs) {
         Process p = null;
         try {
-            p = Runtime.getRuntime().exec(cmdToRunWithArgs);
+            PlatformEnum platform = SystemInfo.getCurrentPlatformEnum();
+            if (platform == PlatformEnum.WINDOWS) {
+                p = Runtime.getRuntime().exec(cmdToRunWithArgs, new String[]{"LANGUAGE=C"});
```

Comment:
Let's make this string array a constant so we don't have to instantiate it every time.